modules/tunnels.sh

```diff
@@ -0,0 +1,72 @@
+function tunnels {
+  checkroot
+  case "$1" in
+    pagekite-install)
+      checkargn $# 3
+      email="$2"
+      sitename="$3"
+      FILE=~/.pagekite.rc
+      check_missing_packages pagekite
+      if [ -f "$FILE" ]; then
+	rm $FILE
```

Comment:
Indentation needs to be corrected.

Reply:
`rm $FILE`

Reply:
if you view the file as a whole it shows the indentation, idk why it looks weird without doing that

Reply:
Right now it's using a tab character. You need to change it to spaces.

Reply:
It's using a tab character. You need to change it to spaces.

Reply:
got it